packages/cli/src/base.ts

```diff
@@ -73,6 +74,13 @@ export abstract class BaseCommand extends LocalCommand {
     if (res.flags && res.flags.privateKey) {
       this._kit.addAccount(res.flags.privateKey)
     }
+    if (res.flags && res.flags.feeCurrency) {
+      const token =
+        res.flags.feeCurrency === 'cUSD' ? CeloContract.StableToken : CeloContract.GoldToken
+      return this._kit.setFeeCurrency(token).then(() => {
```

Comment:
Does this work? I don't think consumers are expecting a promise. Unfortunately it looks like we'd have to update all call sites?

Reply:
hmm I thought this was a hack to get around returning a non-promise but I guess not